build error

with the recent changes in libbcg729 the following problem pops up:

mod_bcg729.c: In function ‘switch_bcg729_init’:
mod_bcg729.c:61:39: error: too few arguments to function ‘initBcg729EncoderChannel’
             context->encoder_object = initBcg729EncoderChannel();

(and a few more)

cheers

--- a/Makefile
+++ b/Makefile
@@ -5,6 +5,7 @@ FS_MODULES=/opt/freeswitch/mod
 ################################
 
 ### END OF CUSTOMIZATION ###
+SHELL := /bin/bash
 PROC?=$(shell uname -m)
 
 CC=gcc 

--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 ################################
-### FreeSwitch headers files ###
-FS_INCLUDES=/opt/freeswitch/include
-FS_MODULES=/opt/freeswitch/mod
+### FreeSwitch headers files found in libfreeswitch-dev ###
+FS_INCLUDES=/usr/include/freeswitch
+FS_MODULES=/usr/lib/freeswitch/mod
 ################################
 
 ### END OF CUSTOMIZATION ###

